embedchain/config/vectordbs/PineconeDbConfig.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional
+
+from embedchain.config.vectordbs.BaseVectorDbConfig import BaseVectorDbConfig
+from embedchain.helper.json_serializable import register_deserializable
+
+
+@register_deserializable
+class PineconeDbConfig(BaseVectorDbConfig):
```

Comment:
The config is quite raw for the first release. More pinecone specific settings like number of replicas can be added to it as the need arises

Reply:
Can we support allow the users to pass **kwargs during init and we pass those args to the pinecone client?